pybullet_toolkit/robots/ur5_rg2.py

```diff
@@ -48,7 +54,7 @@ def pick(self, pos, rot, offset, dynamic=True):
     pre_pos = copy.copy(pos)
     pre_pos[2] += offset
     # rot = pb.getQuaternionFromEuler([np.pi/2.,-np.pi,np.pi/2])
-    pre_rot = pb.getQuaternionFromEuler([0, np.pi, 0])
+    pre_rot = rot
```

Comment:
Typically I don't care about the rotation when doing this positioning. Do you think this way is better?

Reply:
The previous one was actually wrong I guess. The pre grasp pose and grasp pose should have the same rotation